openedx/core/djangoapps/zendesk_proxy/urls.py

```diff
@@ -0,0 +1,11 @@
+"""
+Map urls to the relevant view handlers
+"""
+
+from django.conf.urls import url
+
+from openedx.core.djangoapps.zendesk_proxy.v0.views import ZendeskPassthroughView as v0_view
+
+urlpatterns = [
+    url(r'^v0$', v0_view.as_view(), name='zendesk_proxy_v0'),
```

Comment:
And here we add another line, `url(r'^v1$', v1_view.as_view(), name='zendesk_proxy_v1'),`

This allows for both endpoints to be active simultaneously, so consuming code has time to upgrade before v0 is removed or changed in any way.

Reply:
Also worth calling out - this means the effective url (locally) is now `http://localhost:18010/zendesk_proxy/v0`